### ISSUE TITLE
Metadata and/or thumnails not showing in "Mega Drive - Genesis" Playlist in Ozone

### ISSUE THREAD

**Issue by kazile** [libretro/RetroArch#16124]
Description
I am using RetroArch stable version 1.16.0 with the Ozone interface, and the metadata or thumbnails are not working (they were working fine yesterday). This issue only occurs in the playlist for the "Mega Drive - Genesis" system; it works fine in others. Additionally, in the Ozone interface, XMB, or other menus, I can see the metadata and playlist configurations, which are set to default, same like other playlist which are showing corretly thumnails or metadata when thumbnail is not present.

Expected Behavior
I should be able to see metadata information or thumbnails.

Actual Behavior
Nothing is displayed on the right.

Steps to Reproduce the Bug
Select "Import Content."
Choose manual search.
Enter the zip file extension.
Select the "Mega Drive - Genesis" system.
Choose a specific folder.
Bisect Results
As of January 16, 2024.

Version/Commit
RetroArch: Stable version 1.16.0
Environment Information
OS: Windows 10 64-bit
Compiler: N/A (Not applicable)

Thanks for reading and your time.

**Comment by zoltanvb** [comment]
Thumbnail display and database matching is relying on local files. Have you changed anything regarding the playlist, updated the database (which should not break it, but it would at least give a starting point) or made any change in the thumbnail directory?

**Comment by zoltanvb** [comment]
Actually... Could it be the same as this?
https://github.com/libretro/RetroArch/issues/15805
It is fixed in current nightly builds.

**Comment by kazile** [comment]
It's similar and probably related; it happens with the sixth playlist, but only if you have six playlists. I tried adding a new playlist (for example, PSX), and that fixed it. Then, I deleted the Sega playlist and that new playlist; the PSX playlist had the same problem, the metadata wasn't visible. When I deactivate the netplay option, as in the reported issue, the metadata information appears.

I apologize for reporting a bug that has already been fixed.

**Comment by zoltanvb** [comment]
Can you try installing a recent nightly build, just to make sure? In any case, thank you for taking the time to submit a clear and understandable bug report!

**Comment by kazile** [comment]
Sorry for the late reply. I downloaded RetroArch from https://buildbot.libretro.com/nightly/windows/x86_64/, decompressed the RetroArch.7z file, and replaced the files with the content of RetroArch_update.7z. I'm not sure if it's alright; I don't like to install, and I always use the portable option when I can.

With RetroArch installed as mentioned above, I can see all the metadata from the sixth playlist. I tried adding a seventh, deleting the seventh and the sixth, and everything still works fine.
